tools/homer/test/main.nf

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env nextflow
+
+// Define DSL2
+nextflow.enable.dsl=2
+
+// Log
+log.info ("Starting tests for homer...")
+
+/*------------------------------------------------------------------------------------*/
+/* Define params
+--------------------------------------------------------------------------------------*/
+params.verbose = true
+params.gtf = "$baseDir/input/chr1.gtf"
```

Comment:
these should have been `homer_gtf` - it doesn't matter though because we are able to change the whole parameter structure

Reply:
i.e. params.homer_gtf? or the name of the file?

Reply:
nah the parameters should be prefixed with homer_but we are changing this anyway so don't worry